website/content/docs/connect/ca/vault.mdx

```diff
@@ -136,6 +136,9 @@ The configuration options are listed below.
   must contain a valid chain, where each certificate is followed by the certificate
   that authorized it.
 
+- `RootPKINamespace` / `root_pki_namespace` (`string: <optional>`) - The absolute namespace
+  that the RootPKIPath is in. Setting this overrides the Namespace option for the RootPKIPath. Introduced in 1.12.1
```

Comment:
```suggestion
  that the `RootPKIPath` is in. Setting this overrides the `Namespace` option for the `RootPKIPath`. Introduced in 1.12.1
```
We should use the code font if we are referring to actual configuration options or files, as opposed to the general word for the entity (i.e., _a_ namespace vs. _the_ `namespace` configuration).